install.rdf

```diff
@@ -0,0 +1,18 @@
+<?xml version="1.0" encoding="utf-8"?>
```

Comment:
None of the target applications listed in install.rdf are supported Mozilla products. At least one official Mozilla product must be supported for inclusion on addons.mozilla.org. See https://addons.mozilla.org/firefox/pages/appversions/ for more information on supported target applications on AMO.